void tomo::IoLoop::admit_fd<false, false>(int, tomo::UrKind)
--- PRE

+++ POST

@@ -5,17 +5,17 @@

 000a [ 2] push %r13
 000c [ 2] push %r12
 000e [ 1] push %rbx
-000f [ 3] mov %esi,%r14d
-0012 [ 4] sub $0x30,%rsp
-0016 [ 9] mov %fs:0x28,%rax
-001f [ 5] mov %rax,0x28(%rsp)
-0024 [ 7] mov 0x188(%rdi),%rax
-002b [ 7] movzbl 0x115e0(%rax),%eax
-0032 [ 2] cmp $0x3,%al
-0034 [ 6] ja <+0x430>
-003a [ 7] mov 0x190(%rdi),%rax
-0041 [ 3] mov %edx,%r13d
-0044 [ 3] mov %rdi,%r12
+000f [ 3] mov %rdi,%r12
+0012 [ 3] mov %esi,%r14d
+0015 [ 4] sub $0x30,%rsp
+0019 [ 9] mov %fs:0x28,%rax
+0022 [ 5] mov %rax,0x28(%rsp)
+0027 [ 7] mov 0x188(%rdi),%rax
+002e [ 7] movzbl 0x115e0(%rax),%eax
+0035 [ 2] cmp $0x3,%al
+0037 [ 6] ja <+0x430>
+003d [ 7] mov 0x190(%rdi),%rax
+0044 [ 3] mov %edx,%r13d
 0047 [ 7] mov 0x188(%rdi),%rdx
 004e [ 7] incq 0x450(%rax)
 0055 [ 4] cmp $0x9,%r13b
@@ -90,7 +90,7 @@

 0231 [ 5] mov $0x4000,%edi
 0236 [ 7] movb $0x0,0x780(%rbx)
 023d [ 7] mov %rax,0x630(%rbx)
-0244 [ 7] mov <rip>(%rip),%rax # <tomo_db0::kNoThread+0x680>
+0244 [ 7] mov <rip>(%rip),%rax # <tomo_db0::kNoThread+0x690>
 024b [11] movq $0x0,0x790(%rbx)
 0256 [11] movq $0x0,0x798(%rbx)
 0261 [10] movl $0x0,0x7a0(%rbx)
@@ -101,7 +101,7 @@

 0290 [ 8] vmovdqu %xmm0,0x638(%rbx)
 0298 [ 4] vpxor %xmm0,%xmm0,%xmm0
 029c [ 7] mov %rax,0x784(%rbx)
-02a3 [ 7] mov <rip>(%rip),%rax # <tomo_db0::kNoThread+0x690>
+02a3 [ 7] mov <rip>(%rip),%rax # <tomo_db0::kNoThread+0x6a0>
 02aa [10] vmovdqu8 %zmm0,0x648(%rbx)
 02b4 [10] vmovdqu8 %zmm0,0x688(%rbx)
 02be [10] vmovdqu8 %zmm0,0x6c8(%rbx)
@@ -179,17 +179,16 @@

 0435 [ 9] sub %fs:0x28,%rax
 043e [ 6] jne <+0x727>
 0444 [ 4] add $0x30,%rsp
-0448 [ 2] mov %esi,%edi
-044a [ 1] pop %rbx
-044b [ 2] pop %r12
-044d [ 2] pop %r13
-044f [ 2] pop %r14
-0451 [ 1] pop %rbp
-0452 [ 5] jmp <close@plt>
-0457 [ 9] nopw 0x0(%rax,%rax,1)
+0448 [ 1] pop %rbx
+0449 [ 2] pop %r12
+044b [ 2] pop %r13
+044d [ 2] pop %r14
+044f [ 1] pop %rbp
+0450 [ 5] jmp <tomo::IoLoop::fliplisten_retain(int, tomo::UrKind)>
+0455 [11] data16 cs nopw 0x0(%rax,%rax,1)
 0460 [ 5] mov $0x4040,%ecx
 0465 [ 5] mov $0x24,%edx
-046a [ 7] lea <rip>(%rip),%rsi # <tomo::IoLoop::admit_fd<false, false>(int, tomo::UrKind)::kErr>
+046a [ 7] lea <rip>(%rip),%rsi # <tomo::IoLoop::admit_retained_fd<false, false>(int, tomo::UrKind)::kErr>
 0471 [ 3] mov %r14d,%edi
 0474 [ 5] call <send@plt>
 0479 [ 3] test %rax,%rax
@@ -331,7 +330,7 @@

 0727 [ 5] call <__stack_chk_fail@plt>
 072c [ 5] mov $0x4040,%ecx
 0731 [ 5] mov $0x42a,%edx
-0736 [ 7] lea <rip>(%rip),%rsi # <tomo::IoLoop::admit_fd<false, false>(int, tomo::UrKind)::kDenied>
+0736 [ 7] lea <rip>(%rip),%rsi # <tomo::IoLoop::admit_retained_fd<false, false>(int, tomo::UrKind)::kDenied>
 073d [ 3] mov %r14d,%edi
 0740 [ 5] call <send@plt>
 0745 [ 3] test %rax,%rax
